taxonomies/categories.txt

```diff
@@ -17355,36 +17372,111 @@ fr:Biscuits feuilletés, biscuit feuilleté
 nl:Bladerdeegkoekjes, Bladerdeegkoekje
 
 <en:Flaky biscuits
-en:Palmiers
+en:Palmiers, Palmier, Puff pastry biscuit
 de:Schweinsohren, Schweineohrengebäck, Schweine-Öhrchen, Palmiers
 es:Palmeras, palmeritas
-fr:Palmiers, Palmiers feuilletés
+fr:Palmiers, Palmiers feuilletés, Biscuit sec feuilleté, Palmier
 nl:Krakelingen
 wikidata:en:Q2096475
+ciqual_food_code:en:24659
+ciqual_food_name:en:Biscuit puff pastry
+ciqual_food_name:fr:Biscuit sec feuilleté, type palmier ou autres
+
+<en:Flaky biscuits
+en:Puff pastry biscuit from bakery, Palmier from bakery
+fr:Palmier artisanal
+ciqual_food_code:en:24660
+ciqual_food_name:en:Biscuit ( puff pastry), palmier, from bakery
+ciqual_food_name:fr:Palmier, artisanal
 
 <en:Biscuits
-en:Dry biscuits
+en:Dry biscuits, Biscuit, Cookie
```

Comment:
"Biscuit" cannot be a child of "Biscuits"

Reply:
OK je le retire dans la prochaine version / PR